app/retail/templates/jtbd/connect.html

```diff
@@ -1,7 +1,388 @@
+      </section>
+
+      <section id="connect-with-community">
+        <div class="row border-top-1 border-bottom-1 border-grey">
+          <div class="col-12 col-md-9 p-0 py-5">
+            <h2>Have questions before you start with Gitcoin?</h2>
+            <p class="text-grey-400 font-bigger-2">Connect with the Gitcoin community in Discord</p>
+            <div class="row mt-4 mx-0">
+              <a class="btn btn-primary col-12 col-md-auto p-3 px-5" href="https://discord.com/invite/83BK5z55yB" target="_blank">Join Discord</a>
+            </div>
+          </div>
+          <div class="d-none d-md-inline col-md-3 p-0 align-self-center">
+            <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 245 240" style="width: 80%;"><path d="M104.4 103.9c-5.7 0-10.2 5-10.2 11.1s4.6 11.1 10.2 11.1c5.7 0 10.2-5 10.2-11.1.1-6.1-4.5-11.1-10.2-11.1zM140.9 103.9c-5.7 0-10.2 5-10.2 11.1s4.6 11.1 10.2 11.1c5.7 0 10.2-5 10.2-11.1s-4.5-11.1-10.2-11.1z"/><path d="M189.5 20h-134C44.2 20 35 29.2 35 40.6v135.2c0 11.4 9.2 20.6 20.5 20.6h113.4l-5.3-18.5 12.8 11.9 12.1 11.2 21.5 19V40.6c0-11.4-9.2-20.6-20.5-20.6zm-38.6 130.6s-3.6-4.3-6.6-8.1c13.1-3.7 18.1-11.9 18.1-11.9-4.1 2.7-8 4.6-11.5 5.9-5 2.1-9.8 3.5-14.5 4.3-9.6 1.8-18.4 1.3-25.9-.1-5.7-1.1-10.6-2.7-14.7-4.3-2.3-.9-4.8-2-7.3-3.4-.3-.2-.6-.3-.9-.5-.2-.1-.3-.2-.4-.3-1.8-1-2.8-1.7-2.8-1.7s4.8 8 17.5 11.8c-3 3.8-6.7 8.3-6.7 8.3-22.1-.7-30.5-15.2-30.5-15.2 0-32.2 14.4-58.3 14.4-58.3 14.4-10.8 28.1-10.5 28.1-10.5l1 1.2c-18 5.2-26.3 13.1-26.3 13.1s2.2-1.2 5.9-2.9c10.7-4.7 19.2-6 22.7-6.3.6-.1 1.1-.2 1.7-.2 6.1-.8 13-1 20.2-.2 9.5 1.1 19.7 3.9 30.1 9.6 0 0-7.9-7.5-24.9-12.7l1.4-1.6s13.7-.3 28.1 10.5c0 0 14.4 26.1 14.4 58.3 0 0-8.5 14.5-30.6 15.2z"/></svg>
+          </div>
+        </div>
+      </section>
+
+      <section class="row" id="ready-to-connect">
+        <header>
+          <h2 class="pt-4">Ready to connect?</h2>
+        </header>
+        <div class="d-flex mb-5 pt-2">
+          <div class="col-12">
+            <div class="row">
+              <div class="col-12 col-md-6 p-5 col-gap-border bg-lighter mt-2">
+                <h4 class="py-2 font-weight-normal">Connect through fellowship</h4>
+                <h4 class="py-2 pb-md-5 mb-md-5 font-weight-normal">Join the Kernel fellowship program and receive mentorship and access to a lifelong community.</h4>
+                <a class="col-12 col-md-auto btn btn-primary mt-3 p-3 btn-explore" href="/hackathon-list/" target="_self">Explore Kernel</a>
+              </div>
+              <div class="col-12 col-md-6 p-5 col-gap-border bg-lighter mt-5 mt-md-2">
+                <h4 class="py-2 font-weight-normal">Connect through Fellowship</h4>
+                <h4 class="py-2 pb-md-5 mb-md-5 font-weight-normal">Connect with your peers while working on top open source projects.</h4>
+                <a class="col-12 col-md-auto btn btn-primary mt-3 p-3 btn-explore" href="https://kernel.community/" target="_self">Explore Hackathons</a>
```

Comment:
made this are a little diff focus on flexbox I added some flex utilities to deal better with these situations just using flexbox, check and tell me what do you think

Reply:
Resolved in 084f5a2

Reply:
I'm not too sure, I was using absolute to position the btn so that it would always be aligned regardless of the content wrap. If we use flex like this then it takes the absolute positioned btn out of the flow. Do you want to have a look at how I'm doing it now - I've placed another `<div>` inside the col and put the bg there instead. We can control the margins however we like then 👍